dbt/models/census/schema.yml

```diff
@@ -12,3 +12,135 @@ sources:
 
 models:
   - name: census.vw_acs5_stat
+    columns:
+      - name: percent_race_white
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_other
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_nhpi
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_hisp
+        meta:
+          notes: Counts for hispanic/latino population are complicated. See Census Reporter for more info
+      - name: percent_race_black
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_asian
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_aian
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_mobility_no_move
+        description: Percent of people (older than 1 year) who have not moved in the past 12 months
+        meta:
+          notes: ACS variable B07003_004E / B07003_001E
+      - name: percent_mobility_moved_in_county
+        description: Percent of people (older than 1 year) who moved in county in the past 12 months
+        meta:
+          notes: ACS variable B07003_007E / B07003_001E
+      - name: percent_mobility_moved_from_other_state
+        description: Percent of people (older than 1 year) who moved from another state in the past 12 months
+        meta:
+          notes: ACS variable B07003_013E / B07003_001E
+      - name: percent_income_household_received_snap_past_year
+        description: Percent of households that received SNAP in the past 12 months
+        meta:
+          notes: ACS variable B22003_002E / B22003_001E
+      - name: percent_income_below_poverty_level
+        description: Percent of people below poverty level
+        meta:
+          notes: ACS variable B17001_002E / B17001_001E
+      - name: percent_household_total_occupied_w_sel_cond
+        description: "Selected conditions, including: incomplete plumbing or kitchens, overcrowding, 30% or more of the household income spent on rent or monthly owner costs"
```

Comment:
I wasn't entirely sure whether this description should go in the top-level `description` field or the more hidden `meta.notes` field for this and the variables that follow. What do you think?